test/models/rental_test.rb

```diff
@@ -0,0 +1,37 @@
+require "test_helper"
+require 'pry'
+
+describe Rental do
```

Comment:
You should also have a test that verifies the logic written in the `before_create` block